queries/ocaml/highlights.scm

```diff
@@ -1,146 +1,148 @@
-(module_name) @structure
+; Modules
+;--------
 
-[
-  (module_type_name)
-  (type_constructor)
-  (type_variable)
-  (constructor_name)
-  (tag)
-] @type
-(type_binding (type_constructor) @type)
-
-(exception_definition (constructor_declaration (constructor_name) @exception))
-(application_expression (value_path (value_name) @_raise)
-                        (#eq? @_raise "raise")
-                        (constructor_path) @exception)
-(application_expression (value_path (value_name) @_raise)
-                        (#eq? @_raise "raise")
-                        (parenthesized_expression
-                        (application_expression (constructor_path) @exception)))
-(infix_expression (value_path (value_name) @_raise)
-                  (#eq? @_raise "raise")
-                  (application_expression (constructor_path) @exception))
-
-(attribute_id) @property
-(field_name) @field
+[(module_name) (module_type_name)] @constructor
+
+; Types
+;------
+
+(
+  (type_constructor) @type.builtin
+  (#match? @type.builtin "^(int|char|bytes|string|float|bool|unit|exn|array|list|option|int32|int64|nativeint|format6|lazy_t)$")
```

Comment:
Maybe you can use the `#contains?` operator here 
```suggestion
  (#contains? @type.builtin "int" "char" "bytes" "string" "float" "bool" "unit" "exn" "array" "list" "option" "int32" "int64" "nativeint" "format6" "lazy_t")
```

Reply:
Contains will match `integer` when you put `int`, isn't an exact match.